app/retail/templates/shared/footer_scripts_lite.html

```diff
@@ -41,6 +41,17 @@
 <script src="{% static "v2/js/lib/jquery.cookie.js" %}"></script>
 <script src="{% static "v2/js/lib/jquery.select2.min.js" %}"></script>
 <script src="{% static "v2/js/lib/moment.min.js" %}"></script>
+  <div id="sb_widget"></div>
+<script src="{% static "v2/js/chat.js" %}"></script>
+<script src="{% static "v2/js/chatwidget.js" %}"></script>
+<script>
```

Comment:
maybe add if document.contxt['github_handle'] then > code so just picking the logged users?

Reply:
agree; that should go in the prod build